### ISSUE TITLE
[Tentative] Mobile API Architecture Review / Cleanup

### ISSUE THREAD

**Issue by marcotuts** [openedx/openedx-app-ios#444]

**Comment by edschema** [comment]
@volodymyr-chekyrta was this work completed with the plugin architecture improvements?

**Comment by volodymyr-chekyrta** [comment]
> [@volodymyr-chekyrta](https://github.com/volodymyr-chekyrta) was this work completed with the plugin architecture improvements?

Not really. This task was about reviewing mobile backend APIs and finding candidates to align or merge with the web APIs so we don’t keep multiplying different endpoints. We haven’t started this work yet, so it wasn’t covered by the plugin architecture improvements.
